linux_os/guide/system/network/network-nftables/package_nftables_removed/rule.yml

```diff
@@ -14,6 +14,8 @@ rationale: |-
 
 severity: medium
 
+platform: package[nftables] and not package[firewalld]
```

Comment:
This rule is only for nftables, regardless of having firewalld installed. Maybe for this rule, the entire platform condition could be removed. It is about removing the package. If the package is not installed, it will report `notapplicable` which sounds not the proper message to me. I think that `pass` would fit better in this case.

Reply:
I agree not the smoothest way to approach this but still the rationale here is that we do not have the nftables package , when we have firewalld enabled. But as you mentioned earlier in most cases we have nftables package as dependency to firewalld , since firewalld implements its rules through nft rules.

Reply:
What do you think about removing this line completely? I believe it is not necessary in this rule.